ecl2df/wellconnstatus.py

```diff
@@ -0,0 +1,100 @@
+"""Exctracts connection status history for each compdat connection that
+is included in the summary data on the form CPI:WELL,I,J,K. One line is
+added to the export every time a connection changes status. It is OPEN when
+CPI>0 and SHUT when CPI=0. The earliest date for any connection will be OPEN,
+i.e a cell can not be SHUT before it has been OPEN. This means that any cells
+that are always SHUT will be excluded.
+
+The output data set is very sparse compared to the CPI summary data.
+"""
+
+import argparse
+import re
+from typing import Any, List, Set, Tuple
+
+import numpy as np
+import pandas as pd
+from ecl.summary import EclSum, EclSumKeyWordVector
+
+from ecl2df import getLogger_ecl2csv
+
+from .common import write_dframe_stdout_file
+
+
+def df(unsmry_file: str) -> pd.DataFrame:
+    """descr"""
+    eclsum = EclSum(unsmry_file, include_restart=False, lazy_load=False)
```

Comment:
Are you able to use `ecl2df.summary.df()` instead of punching libecl directly? That would help future maintenance.

Reply:
Yes, I was wondering about that. I will try.

Reply:
Now I used `smry = summary.df(eclfiles, column_keys="CPI*")` and it made the code a lot simpler.